### DISCUSSION TITLE
In webhook payload now comes an object "schema"

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#2971] [Archive]
I see object `schema` in the webhook payload with all the data, I that needed there?

**Comment by botzill** [comment]
Is this `schema` needed to get via webhook payload?

**Comment by rijkvanzanten** [comment]
Yep, it's there so other services can perform more optimized as they won't have to go and fetch the schema themselves every time

I don't know what you mean by 

> Is this schema needed to get via webhook payload?

**Reply by botzill** [reply]
I see. Ok, is there any way I can remove that to not sent on API call? Due to our system is build, this is redundant info and we don't need to at this point. Can I remove that via custom webhook?

**Reply by 7thkey** [reply]
Hi there, this isn't answered at all, we really need a way to remove the schema object, it is getting so big that our CRM begins to slow down.

**Reply by rijkvanzanten** [reply]
You can use a regular hook to control what exact data is sent out. 
Feel free to create a PR to strip the non-payload data from webhooks, as that sounds like it's what you'd expect webhooks to send 👍🏻
